File-Formats/DocIO/convert-markdown-to-word-document-in-csharp.md

```diff
@@ -0,0 +1,575 @@
+    using (WordDocument document = new WordDocument(docStream, FormatType.Markdown))
+    {
+        //Save as a Word document into the MemoryStream.
+        MemoryStream outputStream = new MemoryStream();
+        document.Save(outputStream, FormatType.Docx);
+        //Save the stream as a file in the device and invoke it for viewing. 
+        Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("MarkdownToWord.docx", "application/msword", outputStream);
+    }
+//Please download the helper files from the following link to save the stream as a file and open the file for viewing in the Xamarin platform.
+//https://help.syncfusion.com/file-formats/docio/create-word-document-in-xamarin#helper-files-for-xamarin
+}
+{% endhighlight %}
+{% endtabs %}
+
+T> You can also save the markdown file as [HTML](https://help.syncfusion.com/file-formats/docio/html), [PDF](https://help.syncfusion.com/file-formats/docio/word-to-pdf) and [Image](https://help.syncfusion.com/file-formats/docio/word-to-image).
+
+## Customize image data
+
+The .NET Word (DocIO) library provides an MdImageNodeVisited event, which is used to customize image data while importing Markdown file. You can implement the logic to customize the image data by using this MdImageNodeVisited event.
+
+The following code example shows how to load image data based on image source path when importing the Markdown files.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+//Create a Word document instance.
+using (WordDocument document = new WordDocument())
+{
+    //Hook the event to customize the image while importing Markdown.
+    document.MdImportSettings.ImageNodeVisited += MdImportSettings_ImageNodeVisited;
+    //Open the Markdown file.
+    document.Open("Input.md");
+    //Save as a Word document.
+    document.Save("Sample.docx");
+}
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET" %}
+'Create a Word document instance.
+Using document As WordDocument = New WordDocument()
+    'Hook the event to customize the image while importing Markdown.
+    document.MdImportSettings.ImageNodeVisited += MdImportSettings_ImageNodeVisited
+    'Open the Markdown file.
+    document.Open("Input.md")
+    'Save as a Word document.
+    document.Save("Sample.docx")
+End Using
+{% endhighlight %}
+
+{% highlight c# tabtitle="UWP" %}
+//Create a Word document instance.
+using (WordDocument document = new WordDocument())
+{
+    //Hook the event to customize the image while importing Markdown.
+    document.MdImportSettings.ImageNodeVisited += MdImportSettings_ImageNodeVisited;
+    //Open the Markdown file.
+    document.Open("Input.md");
+
+    //Save as a Word document to the MemoryStream.
+    MemoryStream stream = new MemoryStream();
+    await document.SaveAsync(stream, FormatType.Docx);
+    //Save the stream as a Word document in local machine.
+    Save(stream, "Sample.docx");
+    //Please refer to the following link to save the Word document in UWP platform
```

Comment:
//Please refer to the following link to save a Word document in the UWP platform.